website/docs/components/table/partials/code/component-api.md

```diff
@@ -1,38 +1,50 @@
 
-- The `Hds::Table::Tr` component is a template-only component. It supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering) but is not eligible to receive interactions (e.g., it cannot have an `onClick` event handler attached directly to it). It can contain `Hds::Table::Th` or `Hds::Table::Td` components.
-- The `Hds::Table::Th` component is a template-only component. It supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering) and can contain interactive elements but is not eligible to receive interactions itself. It‘s unlikely you‘ll need to add interactive elements, as sorting is already provided.
-- The `Hds::Table::Td` component is a template-only component. It supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering) and can contain interactive elements (e.g., `<td><a href="user-info.html">User info</a></td>`) but is not eligible to receive interactions itself (e.g., it cannot have an `onClick` event handler attached directly to it).
+### Table
 
 <Doc::ComponentApi as |C|>
   <C.Property @name="<:head>" @type="named block">
-    This is a named block where the content for the table head (`<thead>`) is rendered.
+    This is a named block where the content for the table head (`<thead>`) is rendered. Note: most consumers are unlikely to need to use this named block directly.
   </C.Property>
   <C.Property @name="<:body>" @type="named block">
     This is a named block where the content for the table body (`<tbody>`) is rendered.
   </C.Property>
   <C.Property @name="model" @type="array">
-    If defined, sets the data source that gets yielded by the `:body` named block.
+    Indicates the data model to be used by the table.
   </C.Property>
   <C.Property @name="columns" @type="array">
-    Use an `array` hash to define your table columns. While `key` and `label` are required, other options include `isSortable`, `align` (for text-alignment), and `width`.
+  Use a `hash` within the array to define each column.
+    <Doc::ComponentApi as |D|>
+      <D.Property @name="key" @type="string">
+      The column’s key; required if the column is sortable.
+      </D.Property>
+      <D.Property @name="label" @type="string" @required="true">
+      The column’s label; supports internationalization.
+      </D.Property>
+      <D.Property @name="isSortable" @type="boolean" @values={{array "false" "true" }} @default="false">
+      If set to `true`, indicates that a column should be sortable.
+      </D.Property>
+      <D.Property @name="sortingFunction" @type="function">
+      Callback function to provide support for a custom callback.
+      </D.Property>
+      <D.Property @name="align" @type="enum" @values={{array "left" "center" "right" }} @default="left">
+        If set, determines the text alignment for the column.
```

Comment:
This reminds me that we should explain somewhere (probably in the "how to use") that the alignment in the cells works only for the text; if there are other elements (eg. icons with text, or buttons, etc) they need to use alternative methods (eg a flexbox container)

Notice: in a separate task/ticket, not here.